src/components/blocks/Header/style.css

```
@@ -19,3 +23,8 @@
   background-color: #444;
   cursor: pointer;
 }
+
+.header ul li a:hover {
```

Лучше не прописывать такие цепочки. Сложно будет переопределять. Гораздо лучше и правильнее написать класс для `a`

`<a href='#'>Starships</a>` --> `<a href='#' className="link">Starships</a>`

И потом использовать
```suggestion
.link:hover {...}
```

---

исправлю